src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,39 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ProfileServlet extends HttpServlet {
+
+    /**
+     * Store class that gives access to Users.
+     */
+    private UserStore userStore;
+
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        setUserStore(UserStore.getInstance());
+    }
+
+    /**
+     * Sets the UserStore used by this servlet. This function provides a common setup method for use
+     * by the test framework or the servlet's init() function.
+     */
+    void setUserStore(UserStore userStore) {
+        this.userStore = userStore;
+    }
+
+    @Override
```

Comment:
Even though right now all it does is forward the request to the JSP file, I'd suggest adding a preliminary method description.

Reply:
Got it!